MigHerCas/be-my-creator-next
# Typography configuration

## Summary
Inspired by: https://musk-template.webflow.io/utility/style-guide

Typography configurations have been added to the project

## Details
* Explored ChakraUI theming logic
* Duplicated ChakraUI theme object in order to gain visibility 
* Extended `Heading` theme component with different `sizes`
* Extended `Text` theme component with different `sizes`

## Testing
* Ensure consistency in different browser rendering 
* Visually test sizes and look&feel
* Ensure font assets to be correctly downloaded and imported 

## Checklists

- [ ] Tested in local (Frontend: Chrome, Safari, Firefox, Mobile)
- [x] I reviewed my own Pull Request commit by commit
- [x] I didn't just select everything, this PR really does abide by these ^

## Screenshots
![image](https://user-images.githubusercontent.com/29380502/208691239-9728cd43-dc76-4971-b3ff-9997eaa8dc99.png)
